client/components/date-picker/index.jsx

```diff
@@ -203,24 +215,39 @@ class DatePicker extends PureComponent {
 			);
 		}
 
+		if ( hasDateRange ) {
+			modifiers[ 'range-start' ] = this.props.selectedDays.from;
+			modifiers[ 'range-end' ] = this.props.selectedDays.to;
+			modifiers.range = {
+				from: this.props.selectedDays.from,
+				to: this.props.selectedDays.to,
+			};
+		}
+
+		const rootClassNames = classNames( {
+			'date-picker': true,
+			'date-picker--has-range': hasDateRange,
+			[ `date-picker--${ this.props.numberOfMonths }up` ]: true,
+		} );
+
 		return (
 			<DayPicker
 				modifiers={ modifiers }
-				className="date-picker"
-				selectedDays={ this.props.selectedDays }
+				className={ rootClassNames }
 				disabledDays={ this.props.disabledDays }
-				fromMonth={ this.props.fromMonth }
 				month={ this.props.calendarViewDate }
+				fromMonth={ this.props.fromMonth }
+				toMonth={ this.props.toMonth }
 				onDayClick={ this.setCalendarDay }
-				onDayTouchStart={ this.setCalendarDay }
```

Comment:
Are we losing anything here by removing these props?

Reply:
This is one for @enejb to help with.

Please see https://github.com/Automattic/wp-calypso/pull/27287/commits/406a5f6226761b5ef3ff17f35f5d42956880ea6e

Essentially if we leave this in, each time you "touch" a day it triggers the `setCalendarDay` multiple times because the event fires for both touch and click. 

The commit isn't 100% about why this was added apart from "for a11y reasons". However we know click event are fired on touch devices so without a clear reasoning I'm inclined to say it needs to be removed otherwise the range selection logic doesn't work.

Reply:
Got ya, thanks for the explanation :)